examples/test_corpus_dir/corpus_readme

```diff
@@ -0,0 +1 @@
+This directory is used to test fuzzing_corpus on copying a directory.
```

Comment:
Is the goal to have this readme file as part of the corpus? The contents of this file is also bound to become stale in case other example rules start using this directory for other purposes.

Reply:
Delete this content and rename this file to `corpus_0.txt`, of which the name is the same as the other corpus file, to test name conflict.